src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball ball;
+        for (int i = 0; i < 3; i++) {
+            ball = lottery.getRandomBall();
```

Comment:
its not required to do that in Java
```suggestion
        for (int i = 0; i < 3; i++) {
            Ball ball = lottery.getRandomBall();
```